types/stripe-js/embedded-checkout.d.ts

```diff
@@ -0,0 +1,10 @@
+export interface StripeEmbeddedCheckoutOptions {
```

Comment:
Should we also 
```ts
export * from './embedded-checkout';
```
at https://github.com/stripe/stripe-js/blob/master/types/stripe-js/index.d.ts so users can access it in their code?

Reply:
yes!